packages/extension-drag-handle-react/src/DragHandle.tsx

```diff
@@ -30,36 +36,41 @@ export const DragHandle = (props: DragHandleProps) => {
   const [element, setElement] = useState<HTMLDivElement | null>(null)
   const plugin = useRef<Plugin | null>(null)
 
+  const onNodeChangeRef = useRef(onNodeChange)
```

Comment:
I'm not sure about those changes.

I'd assume that the developer integrating the DragHandle has to properly memoize the props passed to our components.

The re-rendering problems occur because those options - when provided - will never deepEqual as they will technically always be new objects or functions.

What do you think @Aslam97 ?

Reply:
@svenadlung what's your thought on this. Cleaner code but more overhead for users or more code to handle unmemoized code from users?

Reply:
This PR moved to https://github.com/ueberdosis/tiptap/pull/6582 @bdbch @svenadlung